apps/gp2-frontend/src/users/__tests__/UserList.test.tsx

```diff
@@ -48,7 +66,9 @@ const renderUserList = async (
   );
   await waitForElementToBeRemoved(() => screen.queryByText(/loading/i));
 };
-
+afterEach(() => {
+  jest.clearAllMocks();
```

Comment:
I think this should be resetAllMocks. If you change it a test will fail, I think this is because it's using a mock set up in another test.